zulipterminal/ui_tools/views.py

```diff
@@ -932,15 +932,15 @@ def keypress(self, size: urwid_Size, key: str) -> str:
 
 
 class StreamInfoView(PopUpView):
-    def __init__(self, controller: Any, color: str,
-                 name: str, desc: str) -> None:
-        # Add 4 (for 2 Unicode characters on either side) to the popup title
-        # length to make sure that the title gets displayed even when the
-        # content is shorter than the title length (+4 Unicode characters).
-        self.width = max(len(desc), len("# {}".format(name)) + 4) + 2
-        description = urwid.Text(desc, align='center')
-        self.height = description.rows((self.width, ))
-        stream_info_content = [description]
+    def __init__(self, controller: Any, color: str, desc: str,
+                 popup_title: str) -> None:
+        stream_info_content = [('', [desc])]
+        popup_width, column_widths = self.table_widths(stream_info_content,
+                                                       len(popup_title))
+        self.width = popup_width
+        stream_info_content = self.make_table_with_categories(
+            stream_info_content, column_widths)
+        self.height = self.get_popup_height(stream_info_content, popup_width)
```

Comment:
Do we actually need to use this here? I suppose we will need it if we enhance this popup.